libs/core/src/components/pds-popover/pds-popover.scss

```diff
@@ -0,0 +1,160 @@
+:host {
+  --background-color: #fff;
+  --box-shadow: 0 8px 14px rgba(0,0,0,0.16), 0 0 4px rgba(0,0,0,0.08);
+  --color: var(--pine-color-neutral-charcoal-400);
+  --arrow-size: 6px;
+  --arrow-offset: 14px;
+
+  --overlay-border-radius: 10px;
+  --overlay-font-size: 14px;
+  --overlay-line-height: 20px;
+  --overlay-min-width: 240px;
+  --overlay-padding: 8px 12px;
```

Comment:
`--pine-spacing-sm` spacing token can be used for 8px. 

Not a blocker but I have used `calc` in a few places where there is not a single token. e.g - `calc(var(--pine-spacing-md) / 2)` could be used for the 12px. Again may be overkill.

Reply:
I didn't want to go the calc route as I wanted it to be explicit when a value doesn't match the tokens. I do believe this can be something we revisit as to not have px values tho.

Reply:
I've always wondered how we wanted to tackle this and would love to have a discussion on it.